chatter.js

```diff
@@ -1,7 +1,8 @@
 Chatter = {
   options: {
     messageLimit: 30,
-    nickProperty: "username",
+    roomLimit: 15,
```

Comment:
I removed this. Now only a set number of rooms are loaded initially, and all rooms loaded after user clicks on "show more button". This set number of initially loaded rooms should be customizable through chatter options.